vendor/TraceKit/tracekit.js

```diff
@@ -396,7 +396,7 @@ TraceKit.computeStackTrace = (function computeStackTraceWrapper() {
 
         for (var i = 0, j = lines.length; i < j; ++i) {
             if ((parts = chrome.exec(lines[i]))) {
-                var isNative = parts[2] && parts[2].indexOf('native') !== -1;
+                var isNative = parts[2] && parts[2].indexOf('native') === 0; // start of line
```

Comment:
This is covered by the `CHROME_48_BLOB` test:

https://github.com/getsentry/raven-js/blob/master/test/vendor/fixtures/captured-errors.js#L327

I've also verified the format is the same in my Chrome 56 (on the console):

```javascript
setTimeout(function callback() {
  [1].forEach(function iterate() { oops(); });
});
```

Emits:

```
Uncaught ReferenceError: oops is not defined
    at iterate (<anonymous>:2:36)
    at Array.forEach (native)
    at callback (<anonymous>:2:7)
```